WordPress/WordPressTest/PostEditorStateTests.swift

```diff
@@ -1,188 +0,0 @@
-import XCTest
-@testable import WordPress
-
-class PostEditorStateTests: XCTestCase {
```

Comment:
These tests were covering what would be typically considered "configuration": the logic in the unit tests is as complex (or simple) as in the code itself, so I decided to remove these.